lib/libpreopen.c

```diff
@@ -187,6 +189,97 @@ po_find(struct po_map* map, const char *path, cap_rights_t *rights)
 	return match;
 }
 
+void 
+po_errormessage(const char *msg){
+	perror(msg);
+  	exit(EXIT_FAILURE);
+}
+int po_create_shmdata(struct po_map* map){
+	int fd,i,r,trailer_len=0,offset=0;
```

Comment:
style(9) prefers that variables not be initialized here, but at the point of use.